bigbio/biodatasets/distemist/distemist.py

```diff
@@ -208,12 +214,17 @@ def _generate_examples(
                 if self.config.schema == "source":
                     entity["concept_codes"] = []
                     entity["semantic_relations"] = []
-                    if filename in linking_file_names:
+                    if self.config.subset_id == "distemist_linking":
                         entity["concept_codes"] = row.code.split("+")
                         entity["semantic_relations"] = row.semantic_rel.split("+")
 
                 elif self.config.schema == "bigbio_kb":
-                    entity["normalized"] = []
+                    if self.config.subset_id == "distemist_linking":
+                        entity["normalized"] = [
+                            {"db_id": code, "db_name": "SNOMED_CT"} for code in row.code.split("+")
+                        ]
+                    else:
+                        entity["normalized"] = [{"db_id": BigBioValues.NULL, "db_name": BigBioValues.NULL}]
```

Comment:
I might be mixed up, but I thought we decided to go with an empty list here `entity["normalized"] = []` 
@leonweber @jason-fries ?

Reply:
I don't think there was a final conclusion on Discord. However, the empty list version causes a unit test to fail:
`Required key 'normalized' does not have any instances`

Is this acceptable?

Reply:
ah, I think I remember now, this is the dataset in which some but not all configs support NED? 

if that is the case then yes, I think the unit test failing for this particular case is OK. 
currently the top level `SUPPORTED_TASKS` indicate "at least one config in the dataset supports this task" but our unit tests don't reflect that distinction at the moment. 

I think we should, 
* make the normalized entries empty lists
* ignore the unit test failure (if that's the only one) and merge this PR 
* short term - separate PR (probably for maintainers) , update the unit tests to check for "any" (as opposed to "all") of the configs matching a supported task
* longer term - separate PR (probably for maintainers), make supported tasks a config level variable/mapping

@jason-fries @leonweber @hakunanatasha

Reply:
@galtay-tempus I updated the script accordingly